cfgov/v1/management/commands/invalidate_single_page.py

```diff
@@ -0,0 +1,23 @@
+from django.core.management.base import BaseCommand, CommandError
+
+from wagtail.contrib.wagtailfrontendcache.utils import purge_url_from_cache
+
+
+class Command(BaseCommand):
+    help = 'Invalidates a page by its full URL'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--url',
+            nargs='?',
+            help='The full URL for the page you wish to invalidate'
+        )
+
+    def handle(self, *args, **options):
+        page_url = options['url']
+        if page_url:
```

Comment:
Instead of defining this as an optional argument (`nargs='?'`) and then checking to see if it's set, how about doing:

```py
def add_arguments(self, parser):
    parser.add_argument('url', help='The full URL for the page you wish to invalidate')

def handle(self, *args, **options):
    page_url = options['url']
```

This will enforce that a URL is passed as an argument to the command, without requiring the `--url`. Optionally you could use `required=True` instead of `nargs='?'` if you prefer to keep the `--url` usage.